eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/equiz.reviewController.js

```diff
@@ -4,15 +4,15 @@ angular.module('equizModule').controller('ReviewController', ReviewController);
     ReviewController.$inject = ['$scope', '$filter', 'reviewDataService'];
```

Comment:
Rename the file according to https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#naming